website/docs/.nvmrc

```diff
@@ -1 +1 @@
-18
```

Comment:
Why did you add this and removed the 18?

Reply:
I use nvm and it was causing issues as my v18's minor version was not meeting the minimum requirement. However lts/hydrogen is also v18 but updated to the latest version so I changed it to make it easier but can revert if needed and I can probably update my v18 to the latest version too.